src/styles/rewiews.scss

```diff
@@ -0,0 +1,16 @@
+.rewiew {
```

Comment:
The class name 'rewiew' is misspelled. It should be 'review'. This could lead to styling issues if the HTML is expecting the correct spelling.